projects/schematics/src/migrations/5_0/methods-and-properties-deprecations/data/methods-and-properties.migration.ts

```diff
@@ -0,0 +1,1790 @@
+import { MethodPropertyDeprecation } from '../../../../shared/utils/file-utils';
+
+export const METHODS_AND_PROPERTIES_DEPRECATIONS_DATA: MethodPropertyDeprecation[] =
+  [
+    {
+      class: 'QuickOrderFormComponent',
+      importPath: '@spartacus/cart/quick-order/components',
+      deprecatedNode: 'globalMessageService',
+      comment:
+        "// TODO:Spartacus - Property 'globalMessageService' was removed from Class 'QuickOrderFormComponent'. ",
```

Comment:
question which would relate to other things here

in the sequence of our migrations, our constructor deprecations removed the dependency, but should we mention additionally that it was removed as a comment in methods-and-properties, where we mentioned the param is gone.

nothing important, but just a question that I have in mind.

Reply:
My question came from thinking about constructors, where we add new dependency, therefore 'new' params, but we don't mention that

Reply:
The constructor doc and schematics already advertise the new construtor params.

As for removed params iin construtors, they will show up as deleted properties in here as well.

Reply:
Yup I understand that, where the deleted properties are added here as well.

Since that is the case, shouldn't we mention the new properties that were added as well (which are the new dependencies for the class)

Reply:
The new properties (that might replace the deleted ones) are meant to be mentioned in the migration comment of the deleted property.
If it's just a new standalone property, it't not in scope of the migration and breaking change handling to list new things.